### ISSUE TITLE
Allow Killaura to target only friends, not all players

### ISSUE THREAD

**Issue by VincentCCandela** [ImpactDevelopment/ImpactIssues#2128]
**Is your feature request related to a problem? Please describe.**
I would like an option added to killaura so that it targets only friends. When killaura targets players, it triggers server anticheats. However, if killaura only targets friends, then I could easily bypass server anticheats.

**Describe the solution you'd like**
Allow killaura to only target friends, not all of the players in the game.

**Describe alternatives you've considered**
Another alternative could be a sort of hit-list for killaura where killaura only hits players who have been in the game since the match starts. This would exclude the bots that fly in at later stages in the game. Furthermore, players should be able to easily add and remove these players on a match-by-match basis. 

**Additional context**
Basically, I am suggesting that players should be able to create some sort of white-list so that killaura only attacks the real players, which the user can verify, so that the server can't send in anti-cheat bots that apppear to client as players but are really just bots. So in conclusion, my suggestion is just a reverse manual option of autocheat which the user can ensure will bypass anticheats.

**Comment by mastalol** [comment]
I'm not sure if this is allowed, I apologize if it isn't.

The OP basically is asking for a "Shit List" hack. You could implement a "Death List" (to keep it age-appropriate) and add an option for "Death List Only" (similar to "Ignore Friends") to the options of kill aura, aimbot, etc.

I really like this idea, never seen it in any other Minecraft client, but I have seen it in hacks for FPS games.
